contributing_to_docs/term_glossary.adoc

```diff
@@ -527,20 +554,22 @@ In addition to "spec file" being allowed related to RPM spec files, general
 usage of "spec" is allowed when describing Kubernetes or OpenShift object specs
 / manifests / definitions.
 
-*Examples of correct usage*
+.Examples of correct usage
 ====
-Update the Pod spec to reflect the changes.
+Update the `Pod` spec to reflect the changes.
```

Comment:
Is it worth adding "spec" up in the example list of alternatives to "object", "resource", etc? Don't need to list every single option, but since it's mentioned down below here...